pkg/restapi/issuer/operation/operations.go

```diff
@@ -223,7 +223,7 @@ func prepareStoreVCRequest(cred []byte, profile string) ([]byte, error) {
 	return json.Marshal(storeVCRequest)
 }
 
-func (c *Operation) getCMSData(tk *oauth2.Token) ([]byte, error) {
+func (c *Operation) getStudentCardCMSData(tk *oauth2.Token) ([]byte, error) {
```

Comment:
There is no need to create separate method and test cases  for this. The only difference is in URL so we should only add new parameter (url) to getCMS data. Please revert coding part, there should only be script  changes for setting up Strapi data fjor travel card.